packages/react-core/src/components/DescriptionList/examples/DescriptionList.md

```diff
@@ -490,3 +490,133 @@ import PlusCircleIcon from '@patternfly/react-icons/dist/js/icons/plus-circle-ic
   </DescriptionListGroup>
 </DescriptionList>
 ```
+
+## Auto fit
+### Auto-fit basic
+```js
+import React from 'react';
+import {
+  Button,
+  DescriptionList,
+  DescriptionListTerm,
+  DescriptionListText,
+  DescriptionListGroup,
+  DescriptionListDescription
+} from '@patternfly/react-core';
+import PlusCircleIcon from '@patternfly/react-icons/dist/js/icons/plus-circle-icon';
+
+<DescriptionList isAutoFit>
```

Comment:
`--pf-c-description-list--GridTemplateColumns--min:250px` doesn't need to bet set on the basic example, it's already set in the CSS.

Reply:
:+1: got it